pkg/health/health_test.go

```diff
@@ -131,7 +131,7 @@ func TestMuxHealthChecker(t *testing.T) {
 		}
 		container.LivenessProbe.HTTPGet.Port = util.NewIntOrStringFromString(port)
 		container.LivenessProbe.HTTPGet.Host = host
-		health, err := mc.HealthCheck("test", "", api.PodStatus{}, container)
+		health, err := mc.HealthCheck("test", "", api.PodStatus{}, container, Timeout)
```

Comment:
Yes, I added a test for the http health checker. Perhaps the tcp and exec checkers would also benefit from tests there.